samples/csharp/end-to-end-apps/ObjectDetection-Onnx/OnnxObjectDetectionWeb/OnnxObjectDetectionWeb.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Both OnnxObjectDetection and the Database integration CI pipeline are failing, but it seems it's because on their build pipeline they explicitly download .net core 3.0, and not 3.1. I did try locally to use .net core 3.1 for onnx detection samples, and it worked locally.  So I'd think what needs to be changed is the build definition. 

https://dev.azure.com/dnceng/public/_build/results?buildId=914931&view=logs&jobId=7a85cf9d-375b-5480-d467-3582578619c4&j=54defcae-a7d2-5172-f52b-3cceb36dfe3e&t=622ae466-5fb6-5ef6-d815-33ac7b334361

I don't know where your CI build definitions are? @briacht

Reply:
It seems the yaml definition is this, and I'd think it would also need to change for the failing tests to run with core 3.1:

https://github.com/dotnet/machinelearning-samples/blob/522fbdba3a865589cc3e17aef7daa9554cd7aeab/.vsts-dotnet-ci.yml#L169